CONTRIBUTING.md

```diff
@@ -153,6 +153,18 @@ get it merged. You can also post in the `#docs` [Slack
 channel](https://knative.slack.com/) to get input on your ideas or find areas to
 contribute before creating a PR.
 
+### Common Troubleshooting issues for PRs
+
+1. CLA fails even though you have signed a CLA. This can happen if someone accepts and commits suggestions due to the email address not matching the address on record for the CLA. 
+
+1. Test Fails. If you do not see a specific error related to a change you made, and instead the errors are related to timeouts then you may want to rerun the test at a later time. There are running tasks that could result in timeouts or rate limiting if your test runs at the same time as the tasks. 
+
+1. Previews are not working - WIP (not sure what's causing this)
+
+1. Cherrypicking - WIP (not an expert on this)
```

Comment:
This doesn't really belong in TS maybe, I don't think it's something we have any automation for - if we don't, could we add it based on labels @evankanderson @mattmoor ? I don't know how to do this yet really, just manual CPs, which like I said doesn't really go here in TS?

Reply:
Automation would be really useful and probably fun to write here.

cc @n3wscott